openedx/core/djangoapps/programs/utils.py

```diff
@@ -565,21 +555,3 @@ def _collect_course_pricing(self, course):
             if seats:
                 self.data['full_program_price'] += float(seats[0]['price'])
             self.data['avg_price_per_course'] = self.data['full_program_price'] / self.data['number_of_courses']
-
-    def _collect_instructors(self, course_run):
-        """
-        Extend the program data with instructor data. The instructor data added here is persisted
-        on each course in modulestore and can be edited in Studio. Once the course metadata publisher tool
-        supports the authoring of course instructor data, we will be able to migrate course
-        instructor data into the catalog, retrieve it via the catalog API, and remove this code.
-        """
-        module_store = modulestore()
-        course_run_key = CourseKey.from_string(course_run['key'])
-        course_descriptor = module_store.get_course(course_run_key)
-        if course_descriptor:
-            course_instructors = getattr(course_descriptor, 'instructor_info', {})
-
-            # Deduplicate program instructors using instructor name
-            self.instructors.update(
-                {instructor.get('name'): instructor for instructor in course_instructors.get('instructors', [])}
-            )
```

Comment:
I'll have to tag @clintonb on this.
As I understood, data would be fetched from the Course Discovery. Program staff members would be used instead of instructors.